### PR TITLE
@ #214 | update the getting started

### PR DESCRIPTION
**Pull request by hoavt**
connects #214

### CODE REVIEW

**Comment by hoatle** [comment]
@hoavt please refer to github issue (@ #xxx) instead for the commit message.

**Comment by hoatle** [comment]
@hoavt error:

```
$ docker-compose exec dev make generate
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.5.1
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 1 source files that are out of date
updating environment: 0 added, 1 changed, 0 removed
reading sources... [100%] getting_started                                                                           
/opt/app/getting_started.rst:168: ERROR: Content block expected for the "note" directive; none found.
/opt/app/getting_started.rst:291: WARNING: Bullet list ends without a blank line; unexpected unindent.
looking for now-outdated files... none found
pickling environment... done
checking consistency... /opt/app/release.rst:: WARNING: document isn't included in any toctree
done
preparing documents... WARNING: html_favicon is not an .ico file
done
writing output... [100%] index                                                                                      
generating indices... genindex
writing additional pages... search
copying static files... WARNING: favicon file 'favicon.png' does not exist
done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 5 warnings.

Build finished. The HTML pages are in _build/html.
```

**Comment by hoavt** [comment]
@hoatle: please review and merge it.

**Comment by hoatle** [comment]
@hoavt still the problem:

```
/opt/app/getting_started.rst:293: WARNING: Bullet list ends without a blank line; unexpected unindent.
```
please make sure to check on local and travis-ci first.

**Comment by hoavt** [comment]
@hoatle please review it and merge

File: docs/getting_started.rst [hunk 1]
```diff
@@ -159,60 +163,76 @@ Check out the video and follow step by step instructions below:
...
 
-Next: :ref:`teracy-dev Git Clone and Vagrant Up <teracy-dev-git-clone-and-vagrant-up>`
 
 .. _teracy-dev-git-clone-and-vagrant-up:
 
 teracy-dev Git Clone and Vagrant Up
 -----------------------------------
 
-1. Open your terminal window and type:
+1. Open your ``Cygwin Terminal`` window **as administrator** and type:
```
Comment (hoatle) [inline docs/getting_started.rst]: wrong

File: docs/getting_started.rst [hunk 2]
```diff
@@ -267,8 +287,19 @@ teracy-dev Git Clone and Vagrant Up
...
           ``vagrant`` to fix this error.
 
-2. Use the ``$ vagrant ssh`` command to access the virtual machine you have just
-   provisioned. You should see the following similar messages:
+      - If you ``$ vagrant up`` but cannot start the VirtualBox, please find "VBoxUSBMon.inf" and
+        "VBoxDrv.inf" in your installation directory then re-install it to fix the issue. The VirtualBox
+        has an installation issue which was reported `here <https://www.virtualbox.org/ticket/4140>`_
+
+
+2. Open another ``Cygwin Terminal`` window **as administrator**, and use the ``$ vagrant ssh`` command to access the virtual machine you have just  provisioned.
```
Comment (hoatle) [inline docs/getting_started.rst]: wrong

**Comment by hoatle** [comment]
need to keep minimal changes as possible, I see lots of unnecessary reformat content

**Comment by hoatle** [comment]
closed, use this instead: https://github.com/teracyhq/dev/pull/236

**Pull request closed without merging**